From 8475fe917aa737c7904fa33df371314f0fa48179 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Thu, 6 Nov 2014 11:37:21 +0100
Subject: [PATCH 2/2] bump build-dependency on icu to >= 52; problem was icu,
 not libc6

This reverts commit 0af472decd4508d58680dda29665ec28c74cbdfc.

Conflicts:
	shlibs.override.libc
---
 changelog            |    4 +++-
 control              |    4 ++--
 rules                |    4 +++-
 shlibs.override.libc |    4 +---
 4 files changed, 9 insertions(+), 7 deletions(-)

diff --git a/changelog b/changelog
index 5fc6389..93d98da 100644
--- a/changelog
+++ b/changelog
@@ -6,6 +6,8 @@ libreoffice (1:4.4.0~beta1~git20141103-1) UNRELEASED; urgency=medium
     - add conditional/includes/libs for system-opencollada
     - use libmariadb-client-lgpl-dev instead of libmariadbclient-dev
       when USE_MARIADB=y
+    - bump build-dependency on icu to >= 52 (see
+      https://bugs.freedesktop.org/show_bug.cgi?id=82229#c38)
   * debian/shlibs.override.icu:
     - update to actual cuurent SOVERSION...
 
@@ -58,7 +60,7 @@ libreoffice (1:4.3.3~rc2~git20141011-1) unstable; urgency=low
     - go sure and make the package depend on libc6/libc6.1/libc0.1 >= 2.19,
       see https://bugs.freedesktop.org/show_bug.cgi?id=82229
 
- -- Rene Engelhard <rene@debian.org>  Sun, 12 Oct 2014 18:55:48 +0200
+ -- Rene Engelhard <rene@debian.org>  Sat, 11 Oct 2014 15:36:58 +0200
 
 libreoffice (1:4.3.2-2) unstable; urgency=medium
 
diff --git a/control b/control
index 8702dff..204ccb2 100644
--- a/control
+++ b/control
@@ -79,7 +79,7 @@ Build-Depends: ant (>= 1.7.0),
                libhunspell-dev (>= 1.1.5-2),
                libhyphen-dev (>= 2.4),
                libice-dev,
-               libicu-dev (>= 4.6),
+               libicu-dev (>= 52),
                libjpeg-dev,
                liblangtag-dev (>= 0.4),
                liblcms2-dev,
@@ -4422,7 +4422,7 @@ Depends: ant (>= 1.7.0),
          libhunspell-dev (>= 1.1.5-2),
          libhyphen-dev (>= 2.4),
          libice-dev,
-         libicu-dev (>= 4.6),
+         libicu-dev (>= 52),
          libjpeg-dev,
          liblangtag-dev (>= 0.4),
          liblcms2-dev,
diff --git a/rules b/rules
index 3aa7c70..5547efd 100755
--- a/rules
+++ b/rules
@@ -168,7 +168,7 @@ SYSTEM_STUFF += vigra
 USE_EXTERNAL_CXXLIBS=y
 SYSTEM_STUFF += mythes
 SYSTEM_STUFF += icu
-ICU_MINVER= (>= 4.6)
+ICU_MINVER= (>= 52)
 SYSTEM_STUFF += librevenge
 SYSTEM_STUFF += libwpd libwpg libwps
 SYSTEM_STUFF += libvisio
@@ -3139,7 +3139,9 @@ endif
 	mv $$TMP debian/uno-libs3/DEBIAN/symbols && \
 	chmod 644 debian/uno-libs3/DEBIAN/symbols
 	dh_installdeb -s
+ifeq "$(BUILD_KFREEBSD)" "y"
 	cat debian/shlibs.override.libc >> debian/shlibs.local
+endif
 ifeq (,$(filter icu, $(SYSTEM_STUFF)))
 	cat debian/shlibs.override.icu >> debian/shlibs.local
 endif
diff --git a/shlibs.override.libc b/shlibs.override.libc
index 2b5c421..0f04116 100644
--- a/shlibs.override.libc
+++ b/shlibs.override.libc
@@ -1,3 +1 @@
-libc 0.1 libc0.1 (>= 2.19)
-libc 6 libc6 (>= 2.19)
-libc 6.1 libc6.1 (>= 2.19)
+libc 0.1 libc0.1 (>= 2.10.2-7)
-- 
1.7.10.4

